db/dbtypes/types.go

```diff
@@ -372,3 +373,16 @@ type BlockDataBasic struct {
 	Time       int64   `json:"time,omitemtpy"`
 	NumTx      uint32  `json:"txlength,omitempty"`
 }
+
+// ChartBlock models data for charts.
+type ChartBlock struct {
+	Height    int64   `db:"height" json:"height"`
+	SBits     float64 `db:"sbits" json:"sbits"`
+	BlockTime int64   `db:"time" json:"time"`
+}
+
+// TicketPrice front charts requested data
+type TicketPrice struct {
+	Price    float64   `json:"price"`
+	DateTime time.Time `json:"date"`
```

Comment:
In line with what I said about intervals, this data only makes sense as a price on a range of blocks.  What is that range?  A StartBlock and EndBlock are needed.